app/src/features/surveys/sampling-information/sites/create/form/SampleSiteGeneralInformationCreateForm.tsx

```diff
@@ -0,0 +1,43 @@
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+import { useFormikContext } from 'formik';
+import { ChangeEvent } from 'react';
+
+/**
+ * Component for adding a name and description to a sampling site on the create form.
+ *
+ * NOTE: When creating a sampling site, the sites are an array and the name and description apply to each of the sites in the array.
+ * A name and description can only be manually entered when creating a single sampling site. If creating multiple,
```

Comment:
question: Is this interaction intended? Or do we want to be able to change the sampling site name and description for all?

Reply:
Renaming multiple in one form submission is ideal. I was trying to keep it simple, but yeah, might be worth saving this feature for when we can rename multiple. We could reuse the control for adding survey areas once it gets merged in - it's in an open PR.